### PR TITLE
docs(build/using-tools): add foundry

### PR DESCRIPTION
**Pull request by tarrencev**

### CODE REVIEW

File: src/docs/developers/build/using-tools.md [hunk 1]
```diff
@@ -46,6 +46,16 @@ module.exports = {
 }
 ```
 
+## Foundry
+
```
Comment (Nicca42) [inline src/docs/developers/build/using-tools.md:50]: Please add a 1/2 liner about what foundry is, including a link to the tool.

**Review by Nicca42 — CHANGES_REQUESTED** [review]
Add description + link

File: src/docs/developers/build/using-tools.md (hunk 1 above)
Comment (qbzzt) [reply src/docs/developers/build/using-tools.md:50]: "Foundry is a blazing fast, portable and modular toolkit for Ethereum application development written in Rust."

File: src/docs/developers/build/using-tools.md [hunk 2]
```diff
@@ -46,6 +46,18 @@ module.exports = {
...
+Foundry is a blazing fast, portable and modular toolkit for Ethereum application development written in Rust.
```
Comment (Nicca42) [inline src/docs/developers/build/using-tools.md:51]: Link to this https://www.paradigm.xyz/2021/12/introducing-the-foundry-ethereum-development-toolbox

As they don't seem to have a real website.

**Review by Nicca42 — CHANGES_REQUESTED** [review]
Looks good

**Review by Nicca42 — APPROVED** [review]

**Pull request merged**